reactioncommerce/reaction
# feat: Add DetailDrawer component for supplementary views

Resolves part of #5157
Impact: **minor**  
Type: **feature**

## Issue

The upcoming Orders panel updates need a Drawer/sidebar to open from the right side of the screen with additional info

## Solution

Create a component that makes opening and managing a side drawer, easy and consistent.

## Breaking changes

none

## Testing
1. Go to `Accounts`
2. Click on `Manage Groups`
3. See the drawer open.
4. Play around with different viewport sizes, including mobile to ensure the drawer still looks good small, and doesn't overlap the primary navigation drawer.
